docs/defi/flow-yield-vaults/cross-chain.md

```diff
@@ -0,0 +1,315 @@
+
+**Security Audit**: Vault must have recent professional audit, proven track record, and established protocol reputation.
+
+**Liquidity**: Sufficient liquidity for deposits/withdrawals, low slippage on rebalancing, and reasonable withdrawal limits.
+
+**Yield Stability**: Consistent yield history, low volatility in APY, and transparent yield source.
+
+**Gas Efficiency**: Efficient deposit/withdraw operations, batching support if available, and reasonable gas costs relative to vault size.
+
+**Integration Compatibility**: Full ERC4626 compliance, no special requirements or restrictions, and standard share accounting.
+
+## Gas Cost Considerations
+
+Cross-chain operations incur gas costs on both Flow and EVM sides:
+
+### Cost Breakdown
+
+**Vault Creation (One-Time):**
+```
+Flow side:
+  - Create strategy: ~0.001 FLOW
+  - Bridge setup: ~0.001 FLOW
+  - Register scheduler: ~0.0005 FLOW
+  Total Flow: ~0.0025 FLOW ($0.002 @ $0.80/FLOW)
+
+EVM side:
+  - Approve vault: ~0.00015 ETH
+  - Initial deposit: ~0.0002 ETH
+  Total EVM: ~0.00035 ETH ($0.70 @ $2000/ETH)
+
+Combined one-time cost: ~$0.70
+```
+
+**Per Rebalance:**
+```
+When no action needed (95%-105%):
+  - Flow: ~0.0001 FLOW ($0.00008)
+  - EVM: $0 (no transaction)
+  Total: ~$0.00008
+
+When rebalancing needed:
+  - Flow bridge: ~0.0005 FLOW ($0.0004)
+  - EVM withdraw: ~0.0002 ETH ($0.40)
+  - EVM burn: ~0.00015 ETH ($0.30)
+  Total: ~$0.70 per rebalance
+```
+
+**Annual Gas Costs (Estimate):**
+```
+Assumptions:
+  - Rebalancing every 60s = 525,600 checks/year
+  - 5% trigger rebalancing = 26,280 rebalances/year
+  - 95% no action = 499,320 no-ops/year
+
+Costs:
+  - No-ops: 499,320 × $0.00008 = $40
+  - Rebalances: 26,280 × $0.70 = $18,396
+
+Total annual gas: ~$18,436
+
+For $100K vault: 18.4% of capital (uneconomical!)
+For $1M vault: 1.84% of capital (marginal)
+For $10M vault: 0.184% of capital (acceptable)
+```
+
+**Conclusion:** Cross-chain strategies are most cost-effective for larger vaults ($1M+).
+
+## Bridge Security Considerations
+
+Using the EVM bridge introduces additional security considerations:
+
+### Bridge Risk Factors
+
+**Escrow Security**: Flow tokens locked in bridge escrow contract must be secure, audited, and monitored. Risk: Escrow hack could drain bridged assets.
+
+**Mint/Burn Integrity**: Bridged tokens must maintain 1:1 backing ratio. Risk: Minting without locking could create unbacked tokens.
+
+**Cross-Chain Timing**: Bridge operations aren't instant (typically 1-2 minutes). Risk: Price movements during bridging.
+
+**EVM Vault Security**: ERC4626 vaults on EVM side have independent security models. Risk: Vault exploit affects bridged assets.
+
+**COA Control**: Cadence contract controls EVM address via COA. Risk: COA vulnerability could compromise EVM assets.
+
+### Mitigation Strategies
+
+**Bridge Audits**: Use only audited, established bridge infrastructure and verify escrow contract security.
```

Comment:
Not sure if this is a good one to highlight because there is only one cross-VM bridge and we created it.